src/main/java/uk/gov/companieshouse/api/accounts/validation/IntangibleAssetsValidator.java

```diff
@@ -610,6 +592,42 @@ private void validatePresenceOfMultipleYearFilerFields(IntangibleAssetsResource
         }
     }
 
+    private boolean validateSubResourceCost(IntangibleAssetsResource resource, String period, Errors errors, IntangibleSubResource intangibleSubResource, boolean isInvalid) {
+
+        isInvalid = false;
```

Comment:
Don't set this to false; we'll only ever want to set this to true in case it was set to false elsewhere

Reply:
Removed that bit of code